lxd/metrics/api_rates.go

```diff
@@ -0,0 +1,93 @@
+package metrics
+
+import (
+	"net/url"
+	"sync/atomic"
+
+	"github.com/canonical/lxd/shared/entity"
+)
+
+// RequestStatus represents the a completed request status category
+type requestStatus string
```

Comment:
requestResult seems more appt? (as the requests have all finished)

Reply:
I agree. I chose 'status' because it is the name of the metric label, should I change it there as well (and on the spec)?